### ISSUE TITLE
🔤 Missing skill definitions

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/design-gc-digital-talent#16]
## 🔤 Copy - Cognitive Accessibility Issue 

There are missing skill definitions for terms that are listed in the skill library on UAT and production

### ❓ Questions 

- @gobyrne, Is there a dependency here on the [Digital Competencies Exercise](https://miro.com/app/board/uXjVNbyg3tA=/) ?  **Not for UAT certainly**
- Will .AAAAA Test skill  be pushed to production?  **A: Skills are not "in the code". They are added by admins through their UI. So no, it will not.** 
- Do we want to add CSS3  as as tool or as a style sheet (programming) language  since we include sas part of the Front-end development skill definition?  **A: It was deliberately not added as a separate skill. I'm ok with that decision.**
- Should all skills titles be capital case?  **A: We've decided everything on our site should be sentence case unless there's a good reason not to**

## ☑ WCAG 2 Criteria:
 - [2.4 Navigable:](https://www.w3.org/WAI/WCAG21/Understanding/navigable) Provide ways to help users navigate, find content, and determine where they are.
 - [ 3.1 Readable:](https://www.w3.org/WAI/WCAG21/Understanding/readable) Make text content readable and understandable.

**List of skills without definitions (e.g. definition reads N/A):** 

- [ ] Accreditation Procedures, Policies, and Practices
- [ ] Application Access Management
- [ ] Application of NIST controls
- [ ] Cryptographic Applications
- [ ] Database Software Installation Processes and Techniques
- [ ] Data Discovery and Profiling
- [ ] Data Movement
- [ ] Digital Product Accessibility
- [ ] Digital Product Prototyping
- [ ] Enhanced Management Framework for IT Projects
- [ ] Enterprise Software
- [ ] Evidence Integrity
- [ ] Graphic Design
- [ ] Government of Canada IT Policies and Standards
- [ ] Government and Department Policies and Standards
- [ ] Implementation of GC Digital Standards in Operations
- [ ] Information Life Cycle
- [ ] Information Privacy Laws
- [ ] Implementation and Enforcement of Policies
- [ ] Integrating GC Digital Standards into Technical Advice
- [ ] Intrusion Technology
- [ ] IT Disaster Recovery Management
- [ ] IT Disaster Recovery Planning
- [ ] IT Infrastructure Management
- [ ] IT Operations Security
- [ ] IT Project Estimating and Planning Techniques
- [ ] IT Project Progress Monitoring
- [ ] IT Software and Hardware Security Requirements
- [ ] Malware Detection
- [ ] Preservation Planning Practices, Policies, and Procedures
- [ ] Root Cause Analysis
- [ ] Security Certification Procedures
- [ ] Security Event Correlation Tools
- [ ] Risk Assessment
- [ ] Root Cause Analysis
- [ ] UX Research Methods
- [ ] Vulnerability Assessment

Programming languages and tools  below are defined with the exact same high-level definition "Apply concrete knowledge of this **product** (language or tool?) to develop a variety of projects." : 

- [ ] .NET  Programming
- [ ] AWS CodeBuild 
- [ ] AWS CodePipeline
- [ ] AWS EC2
- [ ] AWS ECR
- [ ] AWS Lambda
- [ ] AWS RDS
- [ ] AWS S3
- [ ] Adobe XD
- [ ] Angular
- [ ] Apache
- [ ] asp.net
- [ ] Azure Blob Storage
- [ ] Azure Container Registry
- [ ] Azure Pipelines
- [ ] Azure SQL Database
- [ ] C#
- [ ] C++
- [ ] COBOL
- [ ] Docker
- [ ] F# or Visual Basic
- [ ] Git
- [ ] Graph QL
- [ ] HTML
- [ ] Java
- [ ] Javascript
- [ ] Jquery
- [ ] JSP
- [ ] Microsoft Dynamics
- [ ] MongoDB
- [ ] MySQL
- [ ] nginx
- [ ] NodeJS
- [ ] PHP
- [ ] PostgreSQL
- [ ] Python
- [ ] R
- [ ] React
- [ ] Ruby on Rails
- [ ] Sass
- [ ] SharePoint
- [ ] SQL
- [ ] Transact-SQL (T-SQL)
- [ ] Vue

## 🎨 Design File

[Skills library ](https://www.figma.com/file/5LcwA4AmOM8ebdJKuiPs8g/Skills-library?type=design&mode=design&t=mwUTJt7MEx4ZhPOm-0)

## 🙋‍♀️ Proposed Implementation

To address this, Lauren needs to be engaged on how to close the gap. Once there's a plan to address, we can implement in production, then copy skills from prod to UAT.

- Confirm if their are any dependencies on other DTL projects or initiatives
- Research, design and develop 1-2 sentence definitions for each term listed above when the coast is clear. 
 

## 🌎 Localization

[UAT Skill Library](https://uat-talentcloud.tbs-sct.gc.ca/en/applicant/profile-and-applications/skills)

### 📷 Screenshot

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/101275838/aa684b43-e0e1-4f50-be83-93a23098283d)

## ✅ Acceptance Criteria

An  assumption which, when tested, verify that the copy has been added or updated appropriately.

- [ ] All skills in the dialogue menu are defined  for users.

**Comment by gobyrne** [comment]
We should probably grab the production values and then seed them in UAT. @petertgiles is this practical for skills?

**Comment by substrae** [comment]
Sounds like something an API could help with 😱

**Comment by gobyrne** [comment]
We should really do a good skills API. I think gov folks would actually use it.

**Comment by petertgiles** [comment]
@gobyrne Yes, we could definitely extract the skill definitions from prod to update our seeders and run it in UAT.

**Comment by gobyrne** [comment]
Other than the pull from prod to UAT, I don't think this is a dev or design issue. Skills have been worked on a fair bit lately by Lauren and should be in a decent state. Anna's team will be contributing in the future as well.
